From bade08e3f5b7e671eb57ff2b9c6667ed876dbae4 Mon Sep 17 00:00:00 2001
From: Christian Stewart <christian@paral.in>
Date: Wed, 20 Sep 2017 11:41:51 +0100
Subject: [PATCH] odroid xu3, xu4: disable default serial console

This commit disables the default hardcoded ttySAC2 serial console.

This is in an effort to make the serial port less noisy at boot time.

Signed-off-by: Christian Stewart <christian@paral.in>
---
 include/configs/odroid_xu3.h | 4 ++++
 include/configs/odroid_xu4.h | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/include/configs/odroid_xu3.h b/include/configs/odroid_xu3.h
index 40b48f70ab..871a969623 100644
--- a/include/configs/odroid_xu3.h
+++ b/include/configs/odroid_xu3.h
@@ -38,7 +38,11 @@
 
 #define CONFIG_SYS_INIT_SP_ADDR        (CONFIG_SYS_LOAD_ADDR - 0x1000000)
 
+#ifdef ODROID_ENABLE_DEFAULT_CONSOLE
 #define CONFIG_DEFAULT_CONSOLE		"console=ttySAC2,115200n8\0"
+#else
+#define CONFIG_DEFAULT_CONSOLE		"\0"
+#endif
 
 /* USB */
 #define CONFIG_USB_EHCI
diff --git a/include/configs/odroid_xu4.h b/include/configs/odroid_xu4.h
index 29ddaea212..8a81b6e8d7 100755
--- a/include/configs/odroid_xu4.h
+++ b/include/configs/odroid_xu4.h
@@ -113,7 +113,11 @@
 
 #define CONFIG_SYS_INIT_SP_ADDR		(CONFIG_SYS_LOAD_ADDR - 0x1000000)
 
+#ifdef ODROID_ENABLE_DEFAULT_CONSOLE
 #define CONFIG_DEFAULT_CONSOLE		"console=ttySAC2,115200n8\0"
+#else
+#define CONFIG_DEFAULT_CONSOLE		"\0"
+#endif
 
 /* USB */
 #define CONFIG_USB_EHCI
-- 
2.13.5

